scripts/bootstrap.py

```diff
@@ -0,0 +1,65 @@
+# Server Information
+import servers
+
+# SSH Package
+import paramiko
+
+git_dir = "/tmp/dbe"
```

Comment:
I don't think this should be out of root since there's a chance someone might be using a directory of this name for something else. Why don't we just make a `distributed-bitmap-engine` directory right off the bat in home and do all our work there.

Reply:
(not necessarily the same one as the Git repo, mind you)

Reply:
Done.